packages/compass-indexes/src/utils/atlas-search-indexes-link.ts

```diff
@@ -0,0 +1,7 @@
+import type { AtlasClusterMetadata } from '@mongodb-js/connection-storage/renderer';
+
+export function getAtlasSearchIndexesLink({
+  clusterName,
+}: Pick<AtlasClusterMetadata, 'clusterName'>) {
+  return `#/clusters/atlasSearch/${clusterName}`;
```

Comment:
Not sure if cluster names can contain chars that should be encoded for url, but just in case we should probably do it, that's usually a good practice when building urls manually

```suggestion
  return `#/clusters/atlasSearch/${encodeURIComponent(clusterName)}`;
```

Reply:
added in [689a890](https://github.com/mongodb-js/compass/pull/5549/commits/689a890ac7ca7c3f4a3bdc588897a502d8a12356)